appsync/pools/requestMapper.vtl

```diff
@@ -59,12 +59,9 @@
         #elseif($filterSettings.get("contains"))
             #set($expressionTerms = "")
             #foreach($filterSetting in $filterSettings.contains)
-                #set($expressionTerms = "${expressionTerms} :${filter}_c${velocityCount}")
-                #if( $foreach.hasNext ) #set($expressionTerms = "${expressionTerms},")  #end
+                #set($expressionTerms = ":${filter}_c${velocityCount}")
                 #set($ddbDescriber = {"S": $filterSetting})
                 $util.qr($expressionValues.put(":${filter}_c${velocityCount}", $ddbDescriber))
-            #end
-            #if ($expressionTerms != "")
```

Comment:
Could you explain why this change was required? Just trying to wrap my head around this syntax.

Reply:
I'll post the before and after in the main post.